apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/Rates/utils/formatCountdown.test.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import { formatCountdown } from "~/renderer/screens/exchange/Swap2/Form/Rates/utils/formatCountdown";
+
+describe("formatCountdown", () => {
+  it("formats countdown to timer", () => {
+    expect(formatCountdown(1)).toBe("00:01");
+    expect(formatCountdown(10)).toBe("00:10");
+    expect(formatCountdown(60)).toBe("01:00");
+    expect(formatCountdown(61)).toBe("01:01");
+    expect(formatCountdown(-1)).toBe("00:01");
```

Comment:
When would this happen? @cjordan-ledger

Reply:
There is existing logic that makes negative values positive...

https://github.com/LedgerHQ/ledger-live/pull/2313/files#diff-3491da745355daff4fdfd891af2156dfd2fedcfb11db22e49de080e45bf34ac7L51

But looking at the code I think it'll actually be best to remove this as we already check if the countdown is positive before displaying it

Reply:
https://github.com/LedgerHQ/ledger-live/pull/2313/commits/fe514c60dd12e7e2b9a0f63794230e82c9153ad4